### ISSUE TITLE
Git - After using the Simplified Chinese language pack, the GitHub ssh link method will encounter errors

### ISSUE THREAD

**Issue by MemoryCollection** [microsoft/vscode-remote-release#10558]
After cloning the repository file, a reminder will be given to verify the fingerprint. The provided options are "是" and "否" in Chinese. Selecting "Yes" will repeatedly remind whether to confirm the fingerprint.
After uninstalling the Simplified Chinese language pack, it will prompt 'yes' and' no ', and selecting' yes' will result in normal cloning.

The above content was translated using Baidu Translate, and there may be some grammar issues. Please understand

**Comment by TylerLeonhardt** [comment]
@joshspicer I'm getting a feeling that a string comparison isn't done correctly somewhere... like `l10n.t('yes') === 'yes'` which would obviously not be true in a non-English language. Can you take a look?

**Comment by joshspicer** [comment]
I don't think this is in the Remote - SSH extension. From the description, it sounds like the git extension when you clone with an SSH key?

Is that correct @MemoryCollection, if you could grab us a screenshot of the issue that will ensure we fix the right thing 😄
